### DISCUSSION TITLE
[BUG] Remote Login default from elsa 3 and add custom login page

### DISCUSSION THREAD

**Discussion by Thuylinh28** [elsa-workflows/elsa-core#5910] [Q&A]
## Description
Hi I am using elsa-studio to design workflow by elsa-studio-designer.
 i want remove login default from elsa 3 and I my page login . Have any guides ? 

![image](https://github.com/user-attachments/assets/688f8ef0-4d33-425e-afb2-f3b7b2e07c45)

In server config I commented "elsa.UseIdentity" + "elsa.UseDefaultAuthentication" and I add my authen like this :

![image](https://github.com/user-attachments/assets/43971f79-e8ad-43d3-a9ce-cc52806e6c21)

In Web config , I commented "Services.AddLoginModule"
![image](https://github.com/user-attachments/assets/6304f257-456c-45b8-9485-de404c236a78)

**Comment by sfmskywalker** [comment]
Hi! This hasn't been tested yet, but idea of the existing Elsa.Studio.Login module is that itself is an optional module, which means that you should be able to bring your own module. Given that there's currently no documentation around this, you might try and copy this module, rename it and make the necessary changes to make it your own. Now, there might be some stuff in Els.Studio.Login that should be moved out in order to make the login process truly pluggable, but there's only one way to find out, which is to give it a go and see how far you get. And if we need to make any changes to support this scenario, please do not hesitate to report them.

**Reply by Thuylinh28** [reply]
Elsa.Studio.Login  is optional module , so how to remove default login page ? I want to use elsa-studio-designer without login and authentication.
